src/main/java/com/stripe/model/Account.java

```diff
@@ -1117,7 +1261,7 @@ public void setFrontObject(File expandableObject) {
      * verification failed.
      */
     @SerializedName("errors")
-    List<Account.Requirements.Errors> errors;
+    List<Account.Requirements.Error> errors;
```

Comment:
I think this change is a breaking change and is incorrect though you're right that it should have been the name of the class :(